lms/static/scripts/frontend_apps/components/dashboard/OrganizationActivityFilters.tsx

```diff
@@ -33,6 +34,10 @@ export default function OrganizationActivityFilters({
     routes.assignments,
   );
   const students = useAPIFetch<{ students: Student[] }>(routes.students);
+  const studentsWithName = useMemo(
+    () => students.data?.students.filter(s => !!s.display_name),
```

Comment:
This could potentially end up showing names that consist only of whitespace, but I think that's a problem better solved by trimming somewhere on the backend.